src/helpers/MeasureBundleHelpers.ts

```diff
@@ -206,25 +206,51 @@ export function isValidLibraryURL(libraryName: string) {
   return r.test(libraryName);
 }
 
-export function extractLibrariesFromBundle(measureBundle: fhir4.Bundle): {
+/**
+ * Returns the cqls, rootLibIdentifier, and elmJSONs for a collection of libraries
+ * within a Library Bundle
+ */
+export function extractLibrariesFromLibraryBundle(
+  libraryBundle: fhir4.Bundle,
+  rootLibRef: string
+): {
   cqls: ExtractedLibrary[];
   rootLibIdentifier: ELMIdentifier;
   elmJSONs: ELM[];
 } {
-  const measure = extractMeasureFromBundle(measureBundle);
-  const rootLibRef = measure.library[0];
   let rootLibId: string;
   if (isValidLibraryURL(rootLibRef)) rootLibId = rootLibRef;
   else rootLibId = rootLibRef.substring(rootLibRef.indexOf('/') + 1);
 
+  const { cqls, rootLibIdentifier, elmJSONs } = extractLibrariesFromBundle(libraryBundle, rootLibId);
+
+  if (rootLibIdentifier.id === '') {
+    throw new UnexpectedResource('No Root Library could be identified in provided measure bundle');
```

Comment:
I'm seeing GH Actions for the error statement not being covered in a unit test - can we add a unit test for this, similar to the one that we have for `extractLibrariesFromMeasureBundle`?